russian/wardrobe-to-narnia/wardrobe.md

```diff
@@ -0,0 +1,6 @@
+Осторожно наступая на пол из глазури вы медленно, но уверенно двигаетесь к шкафу. 
+С детства знакомый шкаф может унести вас в страну, где пол из камня и твой лучший друг - говорящий лев. 
+Но чуда не произошло, то ли компот был слишком крепок, но в шкафу были толко пустые коробки от зефира, страрые футбольные бутсы Nike и 
+маска для плавания. 
+
+По случаю вы решили [пойти поплавать](https://github.com/udacity/create-your-own-adventure/blob/master/russian/go-to-swim/go-to-swim.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.